Classes/Utility/FLEXKeyboardShortcutManager.m

```diff
@@ -165,9 +165,11 @@ + (void)load
                     pressureLevel++;
                 }
                 if (pressureLevel > 0) {
-                    for (UITouch *touch in [event allTouches]) {
-                        double adjustedPressureLevel = pressureLevel * 20 * touch.maximumPossibleForce;
-                        [touch setValue:@(adjustedPressureLevel) forKey:@"_pressure"];
+                    if (@available(iOS 9.0, *)) {
```

Comment:
you need to add the macro check `FLEX_AT_LEAST_IOS11_SDK` here as well to make it build with CI Xcode 8